src/js/utils/__tests__/ApplicationUtil-test.js

```diff
@@ -9,44 +9,60 @@ describe('ApplicationUtil', function () {
 
   describe('#invokeAfterPageLoad', function () {
 
+    beforeEach(function () {
+      // Clean up application timers.
+      jasmine.clock().uninstall();
```

Comment:
since you call uninstall in the after each maybe you can just call this at the top of the file? or inside the `describe`